null_safety_examples/cookbook/plugins/picture_using_camera/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: package:pedantic/analysis_options.yaml
```

Comment:
I know I'm going to get brick bats for this one, but we should consider migrating flutter projects from `pedantic` to [`flutter_lints`](https://pub.dev/packages/flutter_lints) and dart projects to [`lints`](https://pub.dev/packages/lints)

Reply:
lol brickbats :)

Oki I will try that for this one and the upcoming migrated docs